data/formats-data.js

```diff
@@ -2959,17 +2959,17 @@ exports.BattleFormatsData = {
 		tier: "LC",
 	},
 	sharpedo: {
-		randomBattleMoves: ["protect", "icebeam", "crunch", "earthquake", "waterfall"],
-		randomDoubleBattleMoves: ["protect", "icebeam", "crunch", "earthquake", "waterfall"],
+		randomBattleMoves: ["protect", "icebeam", "crunch", "earthquake", "liquidation"],
```

Comment:
I would keep Waterfall here for the flinch. Every other Liquidation Pokemon is slow.